trailbot_interfaces/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
Let's just keep one license file at the top level

Reply:
done!